examples/integerOffsetFromSet/README.md

```diff
@@ -0,0 +1,3 @@
+Creates two integer fields, loaded from sets.
+
+The integers are linked by the condition that `Second` must be one greater than `First`.
```

Comment:
Could you add a link to the playground from the readme?

The link should be https://finos.github.io/datahelix/playground/#ewogICJmaWVsZHMiIDogWyB7CiAgICAibmFtZSIgOiAiRmlyc3QiLAogICAgInR5cGUiIDogImludGVnZXIiLAogICAgInVuaXF1ZSIgOiBmYWxzZSwKICAgICJudWxsYWJsZSIgOiBmYWxzZQogIH0sIHsKICAgICJuYW1lIiA6ICJTZWNvbmQiLAogICAgInR5cGUiIDogImludGVnZXIiLAogICAgInVuaXF1ZSIgOiBmYWxzZSwKICAgICJudWxsYWJsZSIgOiBmYWxzZQogIH0gXSwKICAiY29uc3RyYWludHMiIDogWyB7CiAgICAiZmllbGQiIDogIkZpcnN0IiwKICAgICJpblNldCIgOiBbMSwgMiwgM10KICB9LCB7CiAgICAiZmllbGQiIDogIlNlY29uZCIsCiAgICAiaW5TZXQiIDogWzMsIDQsIDVdCiAgfSwgewogICAgImZpZWxkIiA6ICJTZWNvbmQiLAogICAgImVxdWFsVG9GaWVsZCIgOiAiRmlyc3QiLAogICAgIm9mZnNldCI6IDEKICB9XQp9IA%3D%3D

Reply:
I will remove this. I feel it doesn't add much as an example - it was more an aid for my testing. The tests are now encapsulated in the unit testing.